src/Symfony/Component/Serializer/NameConverter/CamelCaseToSnakeCaseNameConverter.php

```diff
@@ -19,7 +19,7 @@
  * @author Kévin Dunglas <dunglas@gmail.com>
  * @author Aurélien Pillevesse <aurelienpillevesse@hotmail.fr>
  */
-class CamelCaseToSnakeCaseNameConverter implements NameConverterInterface
+class CamelCaseToSnakeCaseNameConverter implements NameConverterInterface, AdvancedNameConverterInterface
```

Comment:
CamelCaseToSnakeCaseNameConverter was *not* an advanced converter in 7.0 so there is nothing to re-add there, and adding the arguments in the signature natively is a BC break for child classes.

Reply:
Indeed, you're right, plus I didn't get all the history of that PR (https://github.com/symfony/symfony/pull/53898#issuecomment-2011821491).

I'm going to close that PR, thanks!